rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -183,6 +193,7 @@ void COCOMetaDataReader::read_all(const std::string &path)
     {
         if (0 == std::strcmp(key, "images"))
         {
+            int image_id;
```

Comment:
Why is this declared inside the while loop ?
Move it outside

Reply:
The scope of this variable should be limited to when the images info are read from the json so I added the declaration here